pkg/stanza/operator/helper/multiline.go

```diff
@@ -19,102 +19,10 @@ import (
 	"bytes"
 	"fmt"
 	"regexp"
-	"time"
```

Comment:
Can we start with a PR just for moving these parts into flusher.go and splitter.go?

Reply:
ok